acceptance-tests/helpers/awscli/vpcendpoint/vpcendpoint.go

```diff
@@ -0,0 +1,115 @@
+// Package vpcendpoint provides test helpers for setting up an AWS VPC Endpoint
+package vpcendpoint
+
+import (
+	"csbbrokerpakaws/acceptance-tests/helpers/awscli"
+	"fmt"
+)
+
+type CallerIdentity struct {
+	Arn string `json:"Arn"`
+}
+
+type RouteTable struct {
+	RouteTableID string `json:"RouteTableId"`
+}
+
+type RouteTables struct {
+	RouteTables []RouteTable `json:"RouteTables"`
+}
+
+type VpcEndpoint struct {
+	VpcEndpointID string `json:"VpcEndpointId"`
+}
+
+type VpcEndpointResponse struct {
+	VpcEndpoint VpcEndpoint `json:"VpcEndpoint"`
+}
+
+func CreateEndpoint(allowedVPCID, defaultRegion string) string {
+
+	// Get the ARN of the current user
+	getCallerIdentityCommand := []string{
```

Comment:
Personally I prefer to inline these rather than have another variable (that you have to think of a name for), but this approach works too.